### DISCUSSION TITLE
Does LiteLLM support usage tracking for TTS?

### DISCUSSION THREAD

**Discussion by marty-sullivan** [BerriAI/litellm#6699] [Q&A]
I was able to add the tts-1 and tts-hd-1 models via Azure OpenAI to my LiteLLM proxy. It seems to work fine, but I notice there is no usage tracking. Am I missing how to do this or is it basically just a passthrough and there is no way to track API key / Team usage for non-LLM models?

**Comment by marty-sullivan** [comment]
I did some investigation, and it does seem like the model_info for the configured model does have the correct `"input_cost_per_character": 0.00003,` for the tts-hd-1 model. However, it certainly doesn't seem like the cost of these models is showing up in the usage / spend metrics.

So are these parameters just there for nothing or future implementation? Or am I missing where these metrics are actually being tracked?

**Reply by plusye** [reply]
Hi, have you resolved this issue? I’m facing the same problem and would appreciate any insights.

**Reply by marty-sullivan** [reply]
I have not. I have a feeling that, while the parameters are being set for the models, LiteLLM is not currently using them to track usage

**Reply by marty-sullivan** [reply]
Given that there has been no response, and it looks like this might be supported for other platforms like vertex, I've submitted a bug report https://github.com/BerriAI/litellm/issues/7223
